### PR TITLE
deleted var

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Abhiram-Gullapalli** [comment]
@teoli2003 could you review this.

File: files/en-us/web/api/webgl_api/by_example/hello_glsl/index.md [hunk 1]
```diff
@@ -79,19 +79,19 @@ void main() {
...
   if (!(gl = getRenderingContext()))
     return;
 
-  var source = document.querySelector("#vertex-shader").innerHTML;
-  var vertexShader = gl.createShader(gl.VERTEX_SHADER);
+  const source = document.querySelector("#vertex-shader").innerHTML;
+  const vertexShader = gl.createShader(gl.VERTEX_SHADER);
   gl.shaderSource(vertexShader,source);
   gl.compileShader(vertexShader);
   source = document.querySelector("#fragment-shader").innerHTML
```
Comment (teoli2003) [inline files/en-us/web/api/webgl_api/by_example/hello_glsl/index.md:93]:
```suggestion
  source = document.querySelector("#fragment-shader").innerHTML;
```

File: files/en-us/web/api/webgl_api/by_example/raining_rectangles/index.md [hunk 2]
```diff
@@ -66,11 +66,11 @@ button {
...
-  timer,
-  rainingRect,
-  scoreDisplay,
-  missesDisplay;
+let gl;
+let timer;
+let rainingRect;
+let scoreDisplay;
+let missesDisplay;
 function setupAnimation (evt) {
```
Comment (teoli2003) [inline files/en-us/web/api/webgl_api/by_example/raining_rectangles/index.md:74]:
```suggestion
function setupAnimation(evt) {
```

File: files/en-us/web/api/webgl_api/by_example/raining_rectangles/index.md [hunk 3]
```diff
@@ -81,12 +81,12 @@ function setupAnimation (evt) {
   timer = setTimeout(drawAnimation, 17);
   document.querySelector("canvas")
       .addEventListener("click", playerClick, false);
-  var displays = document.querySelectorAll("strong");
+  const displays = document.querySelectorAll("strong");
   scoreDisplay = displays[0];
   missesDisplay = displays[1];
```
Comment (teoli2003) [inline files/en-us/web/api/webgl_api/by_example/raining_rectangles/index.md:86]: We can get rid of the variable using _destructuring_.
```suggestion
  [ scoreDisplay, missesDisplay ] = document.querySelectorAll("strong");
```

File: files/en-us/web/api/webgl_api/by_example/scissor_animation/index.md [hunk 4]
```diff
@@ -68,9 +68,9 @@ button {
 window.addEventListener("load", setupAnimation, false);
 // Variables to hold the WebGL context, and the color and
 // position of animated squares.
-var gl,
-  color = getRandomColor(),
-  position;
+const gl;
```
Comment (teoli2003) [inline files/en-us/web/api/webgl_api/by_example/scissor_animation/index.md:71]: `const` needs an initializer.
```suggestion
let gl;
```

File: files/en-us/web/api/webgl_api/by_example/scissor_animation/index.md [hunk 5]
```diff
@@ -85,8 +85,8 @@ function setupAnimation (evt) {
   // drawing buffer.
   position = [0, gl.drawingBufferHeight];
 
-  var button = document.querySelector("button");
-  var timer;
+  vconst button = document.querySelector("button");
```
Comment (teoli2003) [inline files/en-us/web/api/webgl_api/by_example/scissor_animation/index.md:88]:
```suggestion
  const button = document.querySelector("button");
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/web_audio_api/migrating_from_webkitaudiocontext/index.md [hunk 6]
```diff
@@ -155,7 +155,7 @@ osc.setWaveTable(table);
 var isCustom = (osc.type == osc.CUSTOM); // isCustom will be true
 
 // New standard AudioContext code:
-var osc = context.createOscillator();
+const osc = context.createOscillator();
 osc.type = "sine";       // sine waveform
 osc.type = "square";     // square waveform
 osc.type = "sawtooth";   // sawtooth waveform
```
[the remark this answers is not in the document]
Comment (Abhiram-Gullapalli) [reply files/en-us/web/api/web_audio_api/migrating_from_webkitaudiocontext/index.md:161]: done

**Comment by Abhiram-Gullapalli** [comment]
I should be the one apologizing sir, for the number of mistakes I did.

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
> I should be the one apologizing sir, for the number of mistakes I did.

It is harder than we believe at first sight. That's why we have a review process!

**Pull request merged**